samples/src/main/java/org/corfudb/samples/graph/GraphDB.java

```diff
@@ -0,0 +1,149 @@
+package org.corfudb.samples.graph;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+
+import java.util.*;
+
+/**
+ * Created by shriyav on 5/25/17.
+ */
+
+public class GraphDB {
+    private SMRMap<UUID, Node> vertices;
```

Comment:
For the sake of uniformity use either a vertex or a node?

Also use a map instead of a smrmap type?